setup.py

```diff
@@ -192,5 +192,27 @@ def run(self):
             'pyi-makespec = PyInstaller.utils.cliutils.makespec:run',
             'pyi-set_version = PyInstaller.utils.cliutils.set_version:run',
         ],
-    }
+
+        # For development only -- this should eventually be migrated to another repo, such as ``https://github.com/pyinstaller/pyinstaller-hooks``.
+        'pyinstaller40': [
+            # This entry point refers to a function
+            # that will be invoked with no parameters. It must return a sequence
+            # of strings, each element of which provides an additional absolute
+            # path to search for hooks. This is equivalent to passing the
+            # ``additional-hooks-dir`` command-line option to PyInstaller for
+            # each string in the sequence.
+            'hook-dirs = PyInstaller.loader:hook_dirs',
+
+            # This entry point must return a sequence of
+            # strings, each of which is an absolute path to a directory tree to
+            # pass to pytest for test discovery or an absolute path to a Python
+            # source file to pass to pytest for test discovery.
+            'test = PyInstaller.utils.run_tests:PyInstaller_tests'
```

Comment:
The comment could go into the manual. Thus here we only need something like "Make PyInstaller's hooks tests available, see ... manual".

Reply:
Should this be `test` or `tests` (plural) (I'm in favor of plural)

Reply:
I agree on `tests`. Updated. I'll probably use CodeChat to create a code-readable manual from the hooks repo you created.